<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JanelaCliente</class>
 <widget class="QDialog" name="JanelaCliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ALTERACAO </string>
  </property>
  <widget class="QPushButton" name="pushButton_CONFIRMAR">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>CONFIRMAR</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>212</width>
     <height>108</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>NOME</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_NOME"/>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEdit_TELEFONE"/>
    </item>
    <item row="2" column="2" rowspan="2">
     <widget class="QLineEdit" name="lineEdit_ENDERECO"/>
    </item>
    <item row="3" column="0" rowspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>CPF</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="lineEdit_CPF"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>ENDERECO</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>TELEFONE</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
